docs/DeveloperGuide.md

```diff
@@ -149,103 +149,104 @@ Classes used by multiple components are in the `seedu.storemando.commons` packag
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
 
-#### Proposed Implementation
-
-The proposed undo/redo mechanism is facilitated by `VersionedStoreMando`. It extends `StoreMando` with undo/redo
-history, stored internally as an `storeMandoStateList` and `currentStatePointer`. Additionally, it implements the
-following operations:
-
-* `VersionedStoreMando#commit()` — Saves the current inventory state in its history.
-* `VersionedStoreMando#undo()` — Restores the previous inventory state from its history.
-* `VersionedStoreMando#redo()` — Restores a previously undone inventory state from its history.
-
-These operations are exposed in the `Model` interface as `Model#commitStoreMando()`, `Model#undoStoreMando()`
-and `Model#redoStoreMando()` respectively.
-
-Given below is an example usage scenario and how undo/redo mechanism behaves at each step.
-
-Step 1. The user launches the application for the first time. The `VersionedStoreMando` will be initialized with the
-initial inventory state, and the `currentStatePointer` pointing to that single inventory state.
-
-![UndoRedoState0](images/UndoRedoState0.png)
-
-Step 2. The user executes `delete 5` command to delete the 5th item in the inventory. The `delete` command
-calls `Model#commitStoreMando()`, causing the modified state of the inventory after the `delete 5` command executes to
-be saved in the `storeMandoStateList`, and the `currentStatePointer` is shifted to the newly inserted inventory state.
-
-![UndoRedoState1](images/UndoRedoState1.png)
+#### Reminder Feature
+
+##### Implementation
+
+This portion describes the implementation of the reminder feature which allows users to view items that are expiring 
+a certain number of days which will be given by the user in the command.
+
+1. When the user keys in a command string, `execute` command of the `LogicManager` is called with the given string as input. 
+2. In the method, `LogicManager` calls on the `parseCommand` method of StoreMandoParser to parse the given command.
+3. `StoreMandoParser` parses the string and determines the command given is a `ReminderCommand`. 
+    Then, a `ReminderCommandParser` object is created to further parse the command.
+4. `StoreMandoParser` then calls on the `parse` method of `ReminderCommandParser` to parse the arguments provided.
```

Comment:
I think it would be clearer if we specify "parse the arguments provided in the user command" or something alone those lines.